src/constants/deck.js

```diff
@@ -23,6 +23,7 @@ export const TAGS = {
   BRAWL: 'Brawl',
   EQUALS: 'Equals',
   RUSH: 'Rush',
+  MIDRANGE: 'Midrange',
```

Comment:
```suggestion
  MID_RANGE: 'Mid-range',
```
Shall we write it like this?